build.gradle

```diff
@@ -15,6 +15,8 @@ buildscript {
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     //firebase crashlytics
     classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.2'
+    def nav_version = "2.5.2"
```

Comment:
this isn't needed unless you are importing safe args library in multiple places

Reply:
done